sources/dynamodb/schema_test.go

```diff
@@ -433,21 +530,73 @@ func TestParseIndexes(t *testing.T) {
 		describeTableOutputs: describeTableOutputs,
 	}
 
-	dySchema := schema.Table{Name: "test"}
-
-	err := analyzeMetadata(client, &dySchema)
+	dySchema := common.SchemaAndName{Name: "test"}
+	conv := internal.MakeConv()
+	isi := InfoSchemaImpl{client, 10}
+	indexes, err := isi.GetIndexes(conv, dySchema)
 	assert.Nil(t, err)
 
-	pKeys := []schema.Key{{Column: "a"}, {Column: "b"}}
-	assert.Equal(t, pKeys, dySchema.PrimaryKeys)
```

Comment:
do we have a replacement for this assertion on primary keys ?

Reply:
TestInfoSchemaImpl_GetConstraints covers primary keys assertion.